cloudsmith/CHANGELOG.md

```diff
@@ -1,2 +1,14 @@
 # CHANGELOG - Cloudsmith
 
+### 1.0.0 / 2025-05-01
+
+***Added***:
+
+* Metric support for storage usage, bandwidth usage, token usage, and member activity
+* Event support for security vulnerability scan results, audit logs, policy violations related to vulnerabilities and license compliance
```

Comment:
```suggestion
* Event support for security vulnerability scan results, audit logs, policy violations related to vulnerabilities, and license compliance
```
By adding an Oxford comma, I'm assuming "policy violations related to vulnerabilities" and "license compliance" are two separate list items.